R/error_bars.R

```diff
@@ -38,29 +37,99 @@
 #' 
 #' gs <- gsplot() %>%
 #'    points(1:10, 1:10) %>%
-#'    error_bar(5, 5, y.high=1) 
+#'    error_bar(x=5, y=5, y.high=1, col="green") 
 #' gs 
 error_bar <- function(object, ...) {
   override("gsplot", "error_bar", object, ...)
 }
 
 
-error_bar.gsplot <- function(object, x, y, y.high=0, y.low=0, x.high=0, x.low=0, 
-                             epsilon=0.1, ..., legend.name=NULL, side=c(1,2)){
+error_bar.gsplot <- function(object, ..., legend.name=NULL, side=c(1,2)){
```

Comment:
I feel like the signature for this should be what it was, it is redundant in a way as it is mirrored in the default, but it will help with the argument handling.